Tests.WebView/TestBase.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Threading;
+using NUnit.Framework;
+
+namespace Tests {
+
+    [TestFixture]
+    public abstract class TestBase<T> where T : class, IDisposable, new() {
+
+        private static object initLock = new object();
+        private static bool initialized = false;
+
+        private Window window;
+        private T view;
+
+        protected static string CurrentTestName => TestContext.CurrentContext.Test.Name + Guid.NewGuid(); 
```

Comment:
Guid.NewGuid() .. should not be here .. you really want the test name

Reply:
@Jmales

Reply:
I wasn't able to get the test name here. `TestContext.CurrentContext.FullName` returns the name of the test class as well as `Name` and `MethodName` is null. I also tried giving each test a name without success.

Reply:
MethodName .. no?

Reply:
Read again pls :P MethodName is always null

Reply:
ok .. sorry... thats odd... 
anyway this should not have that guid, second... you may use the guid for the sandboxes... but not here

Reply:
Actually, for some reason, this happens while at asynchronous mode

Reply:
So now we get the test name before the await as you can see in the setup below